src/ui.ts

```diff
@@ -15,6 +15,8 @@ import {
 export type PreInitTranscendAPI = {
   /** Ready event subscriber */
   ready(callback: (transcend: TranscendAPI) => void): void;
+  /** Ready event dispatch queue */
+  readyQueue: ((transcend: TranscendAPI) => void)[];
```

Comment:
I have no idea what this is, but LGTM!

Reply:
haha this repo is part of the effort to open-source our default consent banner! This defines all the basic types used by all the different airgap modules